### ISSUE TITLE
Help Needed - Way to insert char in Pattern Mask

### ISSUE THREAD

**Issue by rgaikar-lilly** [uNmAnNeR/imaskjs#361]
Hey,
I have below pattern
`{
    mask: '00 / 00',
    lazy: true
  }`

Expected Result: when I type two numbers, I want to insert spaces around / automatically without explicitly typing two spaces, i.e. as soon as I type 2nd number, I want / in input box with two spaces surround to it.

Actual Result: when I type two numbers, I need to either type 3rd number or type two space to get the / in the input box.

 So how can I achieve this? Is there any method to update mask value?

**Comment by uNmAnNeR** [comment]
hi! i don't know easy way to do this. `IMask` instance has method `_appendPlaceholder(blockIndex: number)` to add fixed chars. You can check if 2 digits are filled then call it like `_appendPlaceholder(7)`. 7 is just max length of your mask pattern.
But as soon as this is not the first issue on this topic, i'll consider to add some flag in core. So please don't close it.

**Comment by gterras** [comment]
I have a nicely working date mask with space : 

```javascript
const dateformat = 'DD / MM / YYYY'

{
		mask: Date,
		lazy: false,
		overwrite: true,
		autofix: false,
		pattern: 'DD[ ]/[ ]`MM[ ]/[ ]`YYYY',
		min: new Date(1990, 0, 1),
		max: new Date(2199, 0, 1),
		blocks: {
			DD: {
				mask: MaskedRange,
				placeholderChar: 'j',
				from: 1,
				to: 31,
				maxLength: 2,
			},
			MM: {
				mask: MaskedRange,
				from: 1,
				to: 12,
				placeholderChar: 'm',
				maxLength: 2,
			},
			YYYY: {
				mask: MaskedRange,
				from: 1990,
				to: 2199,
				placeholderChar: 'a',
				maxLength: 4,
			},
		},

		format: (value) => dayjs.format(dateformat),
		parse: (value) => dayjs(value, dateformat),
}
``` 

Only (minor) inconvenience is when using keyboard arrows several extra taps are needed to focus on the useful slots.   

Would  a `ignored` mask character solve both of these cases ?

**Comment by uNmAnNeR** [comment]
fixed in 165c1a8d29955fa8e7cfa05029fdcd6b97930fdd with `eager` option
